recipes-fsl/mcore-demos/imx-mcore-demos.inc

```diff
@@ -32,26 +33,26 @@ MCORE_TYPE:mx8ulp-nxp-bsp  = "m33"
 MCORE_TYPE:mx93-nxp-bsp  = "m33"
 MCORE_TYPE:mx95-nxp-bsp  = "m7"
 
-SRC_URI = "${FSL_MIRROR}${SOC}-${MCORE_TYPE}-demo-${PV}.bin;name=${SOC};fsl-eula=true"
+SRC_URI = "${FSL_MIRROR}/${SOC}-${MCORE_TYPE}-demo-${PV}.bin;name=${SOC};fsl-eula=true"
 
 S = "${WORKDIR}/${SOC}-${MCORE_TYPE}-demo-${PV}"
 
-#SRC_URI="https://www.nxp.com/lgfiles/NMG/MAD/YOCTO//imx93-m33-demo-2.14.0.bin;fsl-eula=true;name=imx93"
-
-
 SCR = "SCR-${SOC}-${MCORE_TYPE}-demo.txt"
 
 do_install () {
+    # install mcores binary to /lib/firmware
```

Comment:
This comment is actually wrong, as it installs to /usr/lib/firmware. It's also obvious, so better just to drop the comment